### ISSUE TITLE
[ux] UI audit of dialogs/ context menus in app

### ISSUE THREAD

**Issue by antlam** [mozilla-mobile/focus-android#1260]
Noticing that we're not actively keeping track of the UI here. So I'm filing this to keep things polished and in order. Strictly looking at some padding, color, and type consistencies here. Nothing major.

![screen shot 2017-08-30 at 11 09 35 am](https://user-images.githubusercontent.com/1994820/29982953-964e52b6-8f08-11e7-916c-9855d41aac3d.png)

Moved checklist to eng [meta]: #1659.

**Comment by bbinto** [comment]
UX: S
Eng: S

**Comment by antlam** [comment]
![screen shot 2017-09-08 at 10 59 20 am](https://user-images.githubusercontent.com/1994820/30224744-cc472812-9484-11e7-888e-ddfa26d48a14.png)

Top new, Bottom old. For comparison

**Comment by antlam** [comment]
![spec_dialogs1](https://user-images.githubusercontent.com/1994820/30620468-61691760-9d5a-11e7-9ba5-400c67231760.png)

@pocmo should be the last of it!

**Comment by layely** [comment]
@antlam @bbinto @lime124 is this something that i can work on ?

**Comment by lime124** [comment]
hey @layely! Are you interested in contributing as a designer or an engineer? 

If it's the design side, there are designs posted for this issue but I can totally point you to other issues that need design work. 

If it's engineering, @pocmo would be a better person to ping for what things you should work on.

Thanks for the ping! :)

**Comment by layely** [comment]
@lime124 Sure the engineer side is what I am interested to. Let's see if @pocmo has something to suggest doing here.

**Comment by pocmo** [comment]
@layely Yes, help would be appreciated here. Currently this tracks the current milestone to be released with the next version of Focus. So there's some time pressure. Therefore I'd suggest to create small commits that fix only a specific thing and open separate pull request for them. This way we can merge them separately (and early) and someone can potentially take over later. If you try to fix all of that in a gigantic pull request then this might take too much time (with review).

**Comment by bbinto** [comment]
awesome, thanks @layely

**Comment by layely** [comment]
![2017-10-11-05-59-40](https://user-images.githubusercontent.com/19215803/31442366-bace7034-ae85-11e7-8caa-3ea8b67ea603.png)

![capture - copie](https://user-images.githubusercontent.com/19215803/31442397-d7f9b4ac-ae85-11e7-8e9c-30e6abf54119.PNG)

The first is my capture when I apply changes suggested by @antlam 
The second is @antlam capture

But after applying **#009688 color (100%)** to buttons **ADD** and **CANCEL**, I noticed that their colors differ from @antlam capture. Is there something that I missed here? 

It would be great also if you can give the opacity that you used for the text on the edit_text.

**Comment by mcomella** [comment]
@antlam Additionally, what's the hint text color? It says #FFFFFF in the mock, but it's definitely not white. :)

**Comment by lime124** [comment]
@mcomella it's white with a 40% opacity which will get you that greyish color.

@layely i think that might be the wrong value. try this one: 0A84FF

**Comment by layely** [comment]
> @layely i think that might be the wrong value. try this one: 0A84FF. I'm not sure which text you mean by "edit text". Is that the "Scene.ca" in the mock?

Yes it is. Thank you @lime124 

> Can you give that a go?

 Ok I will post a screenshot here soon.

**Comment by antlam** [comment]
> @layely i think that might be the wrong value. try this one: 0A84FF. I'm not sure which text you mean by "edit text". Is that the "Scene.ca" in the mock?
> 
> Yes it is. Thank you @lime124

Actually this is spec'd out correctly in product already. I didn't spec it here because it doesnt need to be changed. Sorry for the confusion!

**Comment by liuche** [comment]
@mcomella During Sprint planning, we're going through the P2 list. Is this checklist up to date, and could you give a status update?

Relatedly, we can think about how to keep track of contributor work - whether it's enough to give verbal updates, or if we need to comment in Issues, etc.

**Comment by mcomella** [comment]
> During Sprint planning, we're going through the P2 list. Is this checklist up to date, and could you give a status update?

Sorry this was confusing – this issue is from before we changed our Sprint process. I made this the [ux] bug and created a corresponding eng [meta] bug (#1659) with broken out bugs. I made sure that checklist is up-to-date and applied no labels.

> Relatedly, we can think about how to keep track of contributor work - whether it's enough to give verbal updates, or if we need to comment in Issues, etc.

If we break up issues as we agreed to, I think there will generally be comments from contributors in the issue or at least a single, clear PR we could follow. I think this bug is confusing because all the conversations are happening in PRs and checklists don't have a reference to the commit that closed them so it's unclear if they're actually closed or not.

I think the biggest issue with contributors is a clear indication of who a bug is assigned to (and following up to make sure they're still working on it), though I've seen bots for this if we wanted to invest.
